modules/mirror-registry-update.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-installation-images.adoc
+
+[id="upgrading-mirror-registry_{context}"]
+= Upgrading the mirror registry for Red Hat OpenShift
```

Comment:
Is there a reason you didn't use {product-title}?

Reply:
@mburke5678 yeah, the name of the tool itself is "mirror registry for Red Hat OpenShift." It's a terrible name, but just kinda what I'm stuck with!